src/Migrations/M028MovePageHeaderSideBarOptions.php

```diff
@@ -0,0 +1,288 @@
+<?php
+
+namespace P4\MasterTheme\Migrations;
+
+use P4\MasterTheme\MigrationRecord;
+use P4\MasterTheme\MigrationScript;
+
+/**
+ * Add default for Posts page.
+ */
+class M028MovePageHeaderSideBarOptions extends MigrationScript
+{
+    /**
+     * Perform the actual migration.
+     *
+     * @param MigrationRecord $record Information on the execution, can be used to add logs.
+     * phpcs:disable SlevomatCodingStandard.Functions.UnusedParameter -- interface implementation
+     */
+    protected static function execute(MigrationRecord $record): void
+    {
+        $args = array(
+            'post_type' => 'page',
+            'posts_per_page' => -1,
+        );
+
+        $pages = get_posts($args);
+
+        foreach ($pages as $page) {
+            $page_meta_data = get_post_meta($page->ID);
+
+            $page_header_title = $page_meta_data['p4_title'][0] ?? '';
+            $page_header_image = $page_meta_data['background_image'][0] ?? '';
+            $page_header_image_id = $page_meta_data['background_image_id'][0] ?? '';
+            $page_header_description = $page_meta_data['p4_description'][0] ?? '';
+            $page_header_button_title = $page_meta_data['p4_button_title'][0] ?? '';
+            $page_header_button_link = $page_meta_data['p4_button_link'][0] ?? '';
+            $page_header_button_link_checkbox = $page_meta_data['p4_button_link_checkbox'][0] ?? '';
+            $page_header_hide_page_title = $page_meta_data['p4_hide_page_title_checkbox'][0] ?? '';
+
+            // If Sitemap Page, update on Page Title.
+            if ($page->post_title === 'Sitemap') {
```

Comment:
I think it would safer to check post_meta and look for `_wp_page_template` and whether it uses `page-templates/sitemap.php`. Some NROs may have a different page title  in the Sitemap page